README.md

```diff
@@ -20,8 +20,7 @@ source software which enables the use of this currency.
 Pre-Built Binary
 ----------------
 
-For more information, as well as an immediately usable, binary version of
-the Dash Core software, see https://www.dash.org/downloads/.
+For more information read the original Dash whitepaper.
```

Comment:
21210: missing a (dashified) part of it https://github.com/bitcoin/bitcoin/pull/21210/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R6-R10

Reply:
Thanks I have corrected it

Reply:
I think it should be not in chapter "Pre-Build Binary" but in chapter "What is Dash"? 

Probably need to remove these lines:
```
Pre-Built Binary
----------------
```

btw, @UdjinM6 what's is an original Dash whitepaper?

Reply:
> I think it should be not in chapter "Pre-Build Binary" but in chapter "What is Dash"?
> 
> Probably need to remove these lines:
> 
> ```
> Pre-Built Binary
> ----------------
> ```

Good catch!

> btw, @UdjinM6 what's is an original Dash whitepaper?

https://github.com/dashpay/dash/wiki/Whitepaper

but it's quite outdated now... maybe we should mention/link to DIPs here instead